src/Modules/OrchardCore.Commerce/Endpoints/Extensions/ConvertLocalizedHtmlString.cs

```diff
@@ -1,10 +1,12 @@
 using Microsoft.AspNetCore.Mvc.Localization;
 using System.Collections.Generic;
 using System.Linq;
+using SystemEnvironment = System.Environment;
 
 namespace OrchardCore.Commerce.Endpoints.Extensions;
+
 public static class ConvertLocalizedHtmlString
 {
     public static string ConvertLocalizedHtmlStringList(this IReadOnlyList<LocalizedHtmlString> errors) =>
-        string.Join('\n', errors.Select(error => error.Value));
+        string.Join(SystemEnvironment.NewLine, errors.Select(error => error.Html()));
```

Comment:
Why not use `Environment`?

Reply:
OC has it

Reply:
`using Environment = System.Environment;`

Reply:
Not working

Reply:
None of this is necessary. Using fully qualified `System.Environment.NewLine` is appropriate in this scenario.